### PR TITLE
Grails upgrade preparation

### PR DESCRIPTION
**Pull request by klodav**

### CODE REVIEW

File: app/grails-app/views/person/_form.gsp [hunk 1]
```diff
@@ -48,7 +48,7 @@
 <%--
 <div class="field fieldcontain ${hasErrors(bean: personInstance, field: 'roleType', 'error')} ">
     <label for="roleType">
-		${com.k_int.kbplus.RefdataCategory.getByDesc(RDConstants.PERSON_POSITION).getI10n('desc')}
+		${de.laser.RefdataCategory.getByDesc(RDConstants.PERSON_POSITION).getI10n('desc')}
```
Comment (croqueGrec09) [inline app/grails-app/views/person/_form.gsp:51]: qualified reference?

File: app/grails-app/views/person/show.gsp [hunk 2]
```diff
@@ -73,7 +73,7 @@
                         </dl>
 
                         <%--
-                        <dl><dt>${com.k_int.kbplus.RefdataCategory.getByDesc(RDConstants.PERSON_POSITION).getI10n('desc')}</dt>
+                        <dl><dt>${de.laser.RefdataCategory.getByDesc(RDConstants.PERSON_POSITION).getI10n('desc')}</dt>
```
Comment (croqueGrec09) [inline app/grails-app/views/person/show.gsp:76]: qualified reference?

File: app/grails-app/views/subscription/propertiesMembers.gsp [hunk 3]
```diff
@@ -58,9 +58,9 @@
 
     <b>${message(code: 'subscription.propertiesMembers.propertySelected')}: ${filterPropDef?.getI10n('name')}</b>
     <br>${message(code: 'default.type.label')}: ${PropertyDefinition.getLocalizedValue(filterPropDef?.type)}
-    <g:if test="${filterPropDef?.type == 'class com.k_int.kbplus.RefdataValue'}">
+    <g:if test="${filterPropDef?.type == RefdataValue.CLASS}">
         <g:set var="refdataValues" value="${[]}"/>
-        <g:each in="${com.k_int.kbplus.RefdataCategory.getAllRefdataValues(filterPropDef.refdataCategory)}"
+        <g:each in="${de.laser.RefdataCategory.getAllRefdataValues(filterPropDef.refdataCategory)}"
```
Comment (croqueGrec09) [inline app/grails-app/views/subscription/propertiesMembers.gsp:63]: qualified reference?

**Review by croqueGrec09 — COMMENTED** [review]
two remarks

File: app/grails-app/views/person/_form.gsp (hunk 1 above)
Comment (klodav) [reply app/grails-app/views/person/_form.gsp:51]: comment

File: app/grails-app/views/person/show.gsp (hunk 2 above)
Comment (klodav) [reply app/grails-app/views/person/show.gsp:76]: comment

File: app/grails-app/views/subscription/propertiesMembers.gsp (hunk 3 above)
Comment (klodav) [reply app/grails-app/views/subscription/propertiesMembers.gsp:63]: comment

**Review by croqueGrec09 — APPROVED** [review]
approved

**Pull request merged**